src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml

```diff
@@ -75,15 +75,15 @@ limitations under the License.
 
   <struct name="ChargingTargetStruct" apiMaturity="provisional">
     <cluster code="0x0099"/>
-    <item fieldId="0" name="TargetTimeMinutesPastMidnight" type="int16u" min="0" max="1439" default="0"/>
+    <item fieldId="0" name="TargetTimeMinutesPastMidnight" type="int16u" max="1439" default="0"/>
     <item fieldId="1" name="TargetSoC" type="percent" optional="true" default="0"/>
     <item fieldId="2" name="AddedEnergy" type="energy_mwh" min="0" optional="true" default="0"/>
   </struct>
 
   <struct name="ChargingTargetScheduleStruct" apiMaturity="provisional">
     <cluster code="0x0099"/>
-    <item fieldId="0" name="DayOfWeekForSequence" type="TargetDayOfWeekBitmap" min="0x00" max="0x7F"/>
-    <item fieldId="1" name="ChargingTargets" array="true" type="ChargingTargetStruct" length="10"/>
+    <item fieldId="0" name="DayOfWeekForSequence" type="TargetDayOfWeekBitmap" min="0x00" max="0x7F" optional="true"/>
+    <item fieldId="1" name="ChargingTargets" array="true" type="ChargingTargetStruct" length="10" optional="true"/>
```

Comment:
The spec does not list a Conformance for these at all.  It should presumably list O?  Please make sure a spec issue is filed.

Reply:
I have passed this on to specification guy.

Reply:
Thank you.  Please comment here with the issue number once it exists.

Reply:
OK, so that spec PR sets conformance to M for both fields, right?

Reply:
Yes

Reply:
So why are the optional in the XML here?

Reply:
Apologies for that. Optional against those parameters has been removed and files regenerated.

Reply:
I believe this has now been addressed.